src/main/webapp/app/entities/file-upload-exercise/file-upload-exercise.component.html

```diff
@@ -71,14 +71,14 @@ <h4>
                                 <span class="d-none d-md-inline" jhiTranslate="entity.action.edit">Edit</span>
                             </button>
                             <button
-                                type="submit"
-                                (click)="openDeleteFileUploadExercisePopup(fileUploadExercise.id)"
-                                replaceUrl="true"
-                                queryParamsHandling="merge"
-                                class="btn btn-danger btn-sm mr-1"
+                                *jhiHasAnyAuthority="['ROLE_ADMIN']"
```

Comment:
Instructors should also be able to delete file upload exercises

Reply:
You are right. I actually checked that for exercise we know use `isAtLeastInstructor` and `isAtLeastTutor` properties. I updated all the buttons of File Upload Exercise with proper `*ngIf`